vendors/swedbank-pay-sdk-php/README.md

```diff
@@ -0,0 +1,68 @@
+# Swedbank Pay SDK for PHP
+
+![Build status][build-badge]
+[![Latest Stable Version][version-badge]][packagist]
+[![Total Downloads][downloads-badge]][packagist]
+[![Codecov][codecov-badge]][codecov]
+[![License][license-badge]][packagist]
+[![Dependabot Status][dependabot-badge]][dependabot]
+
+[![Swedbank Pay SDK for PHP][og-image]][packagist]
+
+## About
+
+**UNSUPPORTED**: This SDK is at an early stage of development and is not
+supported as of yet by Swedbank Pay. It is provided as a convenience to speed
+up your development, so please feel free to play around. However, if you need
+support, please wait for a future, stable release.
+
+The Swedbank Pay SDK for PHP simplifies integrations against
+[Swedbank Pay's API Platform][api] by providing native PHP interface towards
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)